deploy/charts/rook-ceph/values.yaml

```diff
@@ -7,7 +7,7 @@ image:
   repository: rook/ceph
   # -- Image tag
   # @default -- `master`
-  tag: v1.13.6
+  tag: v1.13.7
```

Comment:
Double checking that we still want to update the helm chart manually

Reply:
Yes for now we will update manually. I wonder if there is a better approach so we can just use the published version so we don't need to include it in the values.yaml, but we can take that up separately.